RevenueCat.xcodeproj/xcshareddata/xcschemes/BackendIntegrationTests.xcscheme

```diff
@@ -89,7 +89,7 @@
          </BuildableReference>
       </BuildableProductRunnable>
       <StoreKitConfigurationFileReference
-         identifier = "../Tests/TestingApps/PurchaseTester/RevenueCat_IntegrationPurchaseTesterConfiguration.storekit">
+         identifier = "../Tests/BackendIntegrationTests/RevenueCat_IntegrationPurchaseTesterConfiguration.storekit">
```

Comment:
the BackendIntegrationTests app was using the StoreKit config file from PurchaseTester UIKit for some reason. I moved it into this app and killed PurchaseTester UIKit